src/transactions/signed/TransferLockedTokens.ts

```diff
@@ -0,0 +1,43 @@
+import { SignedTransaction } from './SignedTransaction'
+import { LockPeriod } from '../../entities/LockPeriod'
+import { validationRules } from '../../utils/validation'
+
+
+export class TransferLockedTokens extends SignedTransaction {
+  public ENDPOINT: string = 'chain/transfer_locked_tokens'
+  public ACTION: string = 'trnsloctoks'
+  public ACCOUNT: string = 'fio.token'
+  public payeePublicKey: string
+  public canVote: number
```

Comment:
boolean

Reply:
done

Reply:
did it just little different, but the signature is boolean now